libcextract/Passes.cpp

```diff
@@ -460,6 +460,16 @@ class FunctionExternalizerPass : public Pass
                         ClangCompat_None, ctx->OFS);
       PrettyPrint::Set_Source_Manager(&ctx->AST->getSourceManager());
 
+      if (ctx->Ibt && ctx->AST) {
```

Comment:
Maybe also check here is Kernel was set? Maybe we should only set Ibt is Kernel was set in the first place...